CHANGELOG.md

```diff
@@ -17,11 +17,13 @@ BUG FIXES:
   values after refresh. [GH-433]
 * `resource/vcd_vapp_vm` `network` block changes caused MAC address changes in existing NICs
   [GH-436,GH-407]
+* Fix a potential data race in client connection caching [GH-453]
```

Comment:
Did this happen only when the cache was enabled? If yes, we could add a mention about that here, so it's clearer that the scope of effect was limited.

Reply:
Yes, indeed. Added.